xrpl/asyncio/wallet/wallet_generation.py

```diff
@@ -31,6 +31,7 @@ async def generate_faucet_wallet(
     wallet: Optional[Wallet] = None,
     debug: bool = False,
     faucet_host: Optional[str] = None,
+    use_case: Optional[str] = None,
```

Comment:
Why not just call this param `user_agent`? `use_case` is a very confusing name IMO

Reply:
We suggested that it be moved to json instead of http header

Reply:
user_agent is specific to xrpl.js - use-case is pointing towards things like: workshop, int test, unit tests..

Reply:
> Should there be a follow up PR which sets `use_case="integration_test"` or `use_case="unit_test"`?

I didn't see a unit test that call the fundWallet function

Reply:
There aren't any

Reply:
> > Should there be a follow up PR which sets `use_case="integration_test"` or `use_case="unit_test"`?
> 
> I didn't see a unit test that call the fundWallet function

No but generate_faucet_wallet is called in places.  That should allow for you pass use case.

Reply:
> user_agent is specific to xrpl.js - use-case is pointing towards things like: workshop, int test, unit tests..

I'm a little confused as to what this means - `user_agent` is in the HTTP header and I think it's a pretty clear name